ros_installation.md

```diff
@@ -0,0 +1,197 @@
+# Install Gazebo and ROS
+
+This document provides an overview about the options to use different versions
+of ROS in combination with different versions of Gz. It is recommended to read
+it before installing the [ros_gz bridge](https://github.com/gazebosim/ros_gz).
+
+# Important! Simple analysis for a quick and correct decision
+
+If you are planning on using a specific version of ROS and don't have a reason
+to use a specific version of Gazebo, you should proceed with the default
+version of Gazebo available from the ROS repository when installing the ros_gz
+bridge:
+
+```
+apt-get install ros-${ROS_DISTRO}-ros-gz
+
+```
+
+This command is going to install the Gazebo libraries corresponding to the
+version detailed below in the section: "Using the default Gazebo version
+for a ROS distribution using binary installations". The rest of this
+document provides more information and other possible installations but
+at this point ROS and Gz are installed and ready.
+
+# Gazebo versions and ROS integration
+
+Gazebo is an independent project like boost, ogre or any other project used by
+ROS. Usually, the latest version of Gazebo available at the beginning of every
+ROS release cycle (for example Gazebo Fortress for ROS2 Humble) is selected as
+the official one to be fully integrated and supported and will be kept during
+the whole life of the ROS distribution.
+
+Note that all Gz libraries policy follows the [semantic
+versioning](http://semver.org/) philosophy for ABI stability, in which all
+versions that have the same major number (`gz-sim7_7.0.0`, `gz-sim7_7.1.0`,
+`gz-sim7_7.0.1`, ...) are binary compatible and thus interchangeable when using
+the same ROS distro.
+
+## Installing Gazebo
+
+### Gz simulator Ubuntu packages
+
+The easiest way of installing Gazebo is to use packages. There are two main
+repositories which host Gz simulator and Gz libraries: one is `packages.ros.org` and the
+other is `packages.osrfoundation.org`. At the time of writing:
+
+ * ***packages.ros.org***
+   * ROS1 Noetic: Gz Citadel
+   * ROS2 Foxy: Gz Citadel
+   * ROS2 Galactic: Gz Edifice
+   * ROS2 Humble: Gz Fortress
+   * ROS2 Rolling: Gz Fortress (changing frequently)
+
+ * ***packages.osrfoundation.org***
+   * Gz Citadel
+   * Gz Edifice
+   * Gz Fortress
+   * Gz Garden
+
+This means that including the osrfoundation repository is not strictly needed
+to get the Gazebo binary package in Ubuntu. It can be installed from the ROS
+repository.
+
+## Using the default Gazebo version for a ROS distribution using binary installations
+
+For the users that need to run a specific version of ROS and want to use all
+the gazebo ROS related packages out-of-the-box, this is the recommended
+section:
+
+### ROS2 Humble and ROS2 Rolling
+ROS2 Humble and ROS2 Rolling support officially the Gazebo Fortress version.
+For a fully-integrated ROS system, Fortress is the recommened version. The way
+to proceed is just to use the ROS repository and do ***not*** use the
```

Comment:
To clarify,  when you say "use" here you specifically mean "download and install binaries from" correct?

Reply:
"use the X repository" I meant ... setup keys and sources.lists for the X repository. So yes, at the end "download and install binaries from X repository".